### PR TITLE
Adfs Module: Improve Inputs and Outputs for the AdfsCertificate Cmdlets

### PR DESCRIPTION
**Pull request by X-Guardian**
This PR improves the descriptions for the `Inputs` and `Outputs` section of the following `AdfsCertificate` cmdlets:

- Add-AdfsCertificate
- Get-AdfsCertificate
- Remove-AdfsCertificate
- Set-AdfsCertificate
- Update-AdfsCertificate

It also removes several explicit references to AD FS 2.0.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/adfs/Add-AdfsCertificate.md [hunk 1]
```diff
@@ -151,14 +151,18 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
 ## OUTPUTS
 
-### None
+### Microsoft.IdentityServer.PowerShell.Resources.ServiceCertificate
+
+Returns the new ServiceCertificate object when the *PassThru* parameter is specified. By default, this cmdlet does not generate any output.
 
 ## NOTES
-* Active Directory Federation Services (AD FS) 2.0 uses certificates for issuing and receiving tokens, publishing federation metadata and communicating through Secure Sockets Layer (SSL).
+* Active Directory Federation Services (AD FS) uses certificates for issuing and receiving tokens, publishing federation metadata and communicating through Secure Sockets Layer (SSL).
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Add-AdfsCertificate.md]:
```suggestion
* Active Directory Federation Services (AD FS) uses certificates for issuing and receiving tokens, publishing federation metadata, and communicating through Secure Sockets Layer (SSL).
```

File: docset/windows/adfs/Get-AdfsCertificate.md [hunk 2]
```diff
@@ -92,10 +92,13 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
 ## INPUTS
 
+### None
+
 ## OUTPUTS
 
 ### Microsoft.IdentityServer.PowerShell.Resources.ServiceCertificate
-This cmdlet generates a class structure that represents the certificate objects for ADFS.
+
+Returns one or more ServiceCertificate objects that represent the certificate objects for ADFS.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Get-AdfsCertificate.md]:
```suggestion
Returns one or more ServiceCertificate objects that represent the certificate objects for AD FS.
```

File: docset/windows/adfs/Remove-AdfsCertificate.md [hunk 3]
```diff
@@ -140,14 +140,19 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+
+A string object is received by the *Thumbprint* parameter.
 
 ## OUTPUTS
 
 ### None
 
 ## NOTES
-* Removing a certificate removes it only from the Active Directory Federation Services (AD FS) 2.0 configuration data. It does not remove or delete the certificate from the local certificate store on the server computer.
+* Removing a certificate removes it only from the Active Directory Federation Services (AD FS) configuration data. It does not remove or delete the certificate from the local certificate store on the server computer.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Remove-AdfsCertificate.md]:
```suggestion
* Removing a certificate removes it only from the AD FS configuration data. It does not remove or delete the certificate from the local certificate store on the server computer.
```

**Comment by e0i** [comment]
@get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by e0i** [comment]
@get-itips Any additional comments?

**Pull request merged**